openapi3.yaml

```diff
@@ -204,6 +204,17 @@ components:
           $ref: '#/components/schemas/status'
       additionalProperties: false
     findRecordModel:
+      type: object
+      description: raster catalog find model
+      properties:
+        metadata:
+          $ref: >-
+            ./Schema/fullLayerMetadata.yaml#/components/schemas/fullLayerMetadata
+        links:
+          type: array
+          items:
+            $ref: ./Schema/link.yaml#/components/schemas/Link
+    findRecordModelRequest:
```

Comment:
Now that we have `findRecordModelRequest` why do we need this duplicate model: `findRecordModel`

Reply:
`findRecordModelRequest`  is used in the find routes as the request body - just changed the name. we want to allow and not break the request type. and allow search by id.
`findRecordModel` is used in the response because we never really returned the id for the /find route so it is not a breaking change